.ci/pipeline/ci.yml

```diff
@@ -36,6 +36,14 @@ jobs:
     inputs:
       artifactName: '$(platform.type) build'
       targetPath: '__release_lnx_gnu'
+  - script: |
+      .ci/scripts/conformance_test.sh --build-dir __release_lnx_gnu
```

Comment:
In this case conformance will be executed for both 32 and 64 bits. Does 32 bits make sense?